automatic/waterfox/update.ps1

```diff
@@ -1,9 +1,15 @@
-﻿import-module au
-
+import-module au
```

Comment:
Looks like you have removed the UTF-8 BOM encoding on this file.
For compatibility reasons, it is best not to remove that on Powershell files.

Reply:
done by mistake